src/assets/scripts/utilities/unitConverters.js

```diff
@@ -270,6 +270,10 @@ export const parseElevation = (elevation) => {
         return parseFloat(elevation);
     }
 
+    if (elevation === 'Infinity' || elevation === '-Infinity')  {
```

Comment:
This is the correct logic, I can see the tests are passing, but I would prefer to combine the logic here with the logic on line 269:
```javascript
if (_isNumber(elevation) || elevation === 'Infinity' || elevation === '-Infinity') {
    return parseFloat(elevation);
}
```

could you also add a comment about why we are looking for strings here?